src/ripple/protocol/Keylet.h

```diff
@@ -95,6 +96,20 @@ static_assert(std::is_move_assignable_v<Keylet>);
 static_assert(std::is_nothrow_destructible_v<Keylet>);
 #endif
 
+template<bool>
+class DPAuthImpl;
+
+struct DPAuthKeylet final : public KeyletBase {
+    template <bool Writable>
+    using TWrapped = DPAuthImpl<Writable>;
+
+    using KeyletBase::check;
+
+    explicit DPAuthKeylet(uint256 const& key) : KeyletBase(ltDEPOSIT_PREAUTH, key)
+    {
+    }
+};
+
```

Comment:
By the time everything is done there will be quite a number of these specialized `Keylet`s in the file.  I think it would be a good idea to (almost) alphabetize the `Keylet`s.  Put `KeyletBase` first, followed by `Keylet`.  Then alphabetize the remaining `Keylet`s as they are added.

Reply:
okay. By alphabetize, you mean capitalize the first letter of the keylet? like `DepPreAuthKeylet` ?

Reply:
> By alphabetize, you mean capitalize the first letter of the keylet? like `DepPreAuthKeylet` ?

Sorry for not being clear.  By the time you are done Keylet.h will contain a bunch of classes derived from `KeyletBase`.  I'm suggesting that the file should keep (most) of those definitions in alphabetical order so they are easier for folks to find.  For example, this file now contains class definitions in this order:
```
KeyletBase
Keylet
DepPreAuthKeylet
AccountRootKeylet
```
I think it makes sense for `KeyletBase` and `Keylet` to stay at the top.  But the remainder should be alphabetical.  So, for this revision of the file they'd be in this order:
```
KeyletBase
Keylet
AccountRootKeylet
DepPreAuthKeylet
```
It's not really important right now.  But by the time you get to the end of the project there will be quite a number of `Keylet`s in here, and they will be easier to find if they are alphabetical.

Make sense?

Reply:
yes okay, I'll keep this in mind 👍

Reply:
yeah, it will be easier to organize them in alphabetical order once all the PRs are merged into once place